app/controllers/admin/preferences_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::PreferencesController < Admin::BaseController
+  before_action :require_admin_owner
+
+  def show
+    @totp_enabled = current_admin.otp_required_for_login
```

Comment:
Wondering whether we can't just get this directly from the admin in the view since it's loaded and this is just a bool. No idea what's better though, since I suppose this is clearer in the controller

Reply:
Based on one of your earlier comments, I think we can get rid of this and do something like `current_admin.totp_enabled?`